src/persist-client/src/impl/state.rs

```diff
@@ -543,28 +543,58 @@ mod tests {
     fn downgrade_since() {
         let mut state = State::<(), (), u64, i64>::new(ShardId::new());
         let reader = ReaderId::new();
+        let reader2 = ReaderId::new();
         let _ = state.collections.register(SeqNo::minimum(), &reader);
+        let _ = state.collections.register(SeqNo::minimum(), &reader2);
         // Greater
         assert_eq!(
             state
                 .collections
                 .downgrade_since(&reader, &Antichain::from_elem(2)),
             Continue(Since(Antichain::from_elem(2)))
         );
+        assert_eq!(state.collections.since, Antichain::from_elem(0));
         // Equal
```

Comment:
these equal and less cases are no longer testing what they're intended to test (the shard-global since)

Reply:
changed the test to add a new reader after these cases complete

Reply:
changed the test to add a new reader after these cases complete